deps/v8/gypfiles/v8.gyp

```diff
@@ -8,7 +8,7 @@
     'v8_random_seed%': 314159265,
     'v8_vector_stores%': 0,
     'v8_embed_script%': "",
-    'v8_extra_library_files%': [],
+    'v8_extra_library_files%': [],  # Placeholder. For upstream V8, this includes test files that Node.js does not need.
```

Comment:
Can you put the comment on its own line and try to keep it < 80 columns?

Reply:
Ack.